docs/Config_Reference.md

```diff
@@ -250,6 +250,11 @@ max_z_velocity:
 #   maximum speed of up/down moves (which require a higher step rate
 #   than other moves on a delta printer). The default is to use
 #   max_velocity for max_z_velocity.
+#max_z_accel:
+#   This sets the maximum acceleration (in mm/s^2) of movement along
+#   the z axis. Setting this may be useful if the printer can reach higher acceleration
+#   on XY moves than Z moves (eg, when using input shaper).  The default
```

Comment:
Would you mind removing the spurious extra spaces?

Reply:
I found one space.. and rearranged it to have a nicer right "border"